Lib/test/test_json/test_recursion.py

```diff
@@ -70,17 +70,18 @@ def default(self, o):
 
     @support.skip_emscripten_stack_overflow()
     def test_highly_nested_objects_decoding(self):
+        very_deep = 200000
```

Comment:
Ah, I actually thought if this value could be shared across multiple tests. I don't know if 200_000 is just something random or if we can say "recursion errors should happen with $N \geq 200000$)". (I would also put the value below the comment, but that's a nit)

Reply:
The numbers differ for different tests as they use different amounts of stack, so each test gets a different number.

The exact numbers don't matter as long as they are large enough to trigger the overflow on all platforms and not so large the test takes ages.